cmd/helm/create.go

```diff
@@ -97,17 +99,22 @@ func (o *createOptions) run(out io.Writer) error {
 		APIVersion:  chart.APIVersionV2,
 	}
 
+	destination := o.directory
+	if destination == "" {
+		destination = o.name
```

Comment:
Don't we want the destination to be `filepath.Dir(o.name)`? That would match the old behaviour.